src/platforms/apple/common/dsym.mdx

```diff
@@ -46,15 +50,18 @@ An App Store Connect user and password are required, in addition to the normal A
 
 We highly recommend that you create a dedicated App Store Connect user account for the purpose of this integration, and that you **not** use a regular user account.
 
-Using the configured App Store Connect account on a different device will disrupt Sentrys ability to use it in order to download dSYM files.
+Using the configured App Store Connect account on a different device will disrupt Sentry's ability to use it in order to download dSYM files.
 
 </Alert>
 
 These credentials will need to be verified by a two-factor authentication token that you can choose to have sent by SMS.
 
 **Re-authentication**
 
-The App Store Connect credentials and two-factor authentication are short-lived and need to be refreshed on a regular basis. Sentry will prompt users to re-authenticate once the App Store Connect credentials expire.
+The App Store Connect credentials and two-factor authentication are short-lived and need to be refreshed on a regular basis. Sentry will prompt users to re-authenticate in the Project Settings side bar as well as the Custom Repositories section under a project's Debug Files settings once their App Store Connect credentials expire.
+
+![Warning to update credentials in sidebar](sidebar-warning.png)
```

Comment:
This seems way to large.  could it be scaled down?

Reply:
Don't show the entire sidebar for this. We've already told the reader where to expect to see this warning, so the screenshot is ultimately about showing them what the warning looks like. I'd cut this down to about 5 items in the menu with Debug Files in the middle.

Reply:
so i can fix this up by simply cropping the existing image, but i can't seem to get the warning to show up any more when the credentials are expired:

<img width="1098" alt="Screenshot 2021-09-07 at 11 09 46 AM" src="https://user-images.githubusercontent.com/5597345/132388039-35e85ed5-9fad-4d18-b664-48bae42df7a3.png">

@flub is this expected behaviour? we can remove everything related to this warning if it's no longer one of the places where we warn the user.

Reply:
this should still be shown so leave it in.  maybe some condition wasn't triggered?  You need to also have pending downloads before it shows up.